### ISSUE TITLE
[ColumnSeries] Only one bar

### ISSUE THREAD

**Issue by LuisCampos97** [syncfusion/flutter-widgets#117]
I have this widget code to show two bars next to each other but as you can see from the image, only one bar appears.

```dart
class SalesMonthCompChart extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    List<ChartData> data = [ChartData(month: 12, value: 16000), ChartData(month: 1, value: 8000)];

    return SizedBox(
        height: 170,
        child: SfCartesianChart(
            plotAreaBorderWidth: 0,
            margin: EdgeInsets.zero,
            enableSideBySideSeriesPlacement: false,
            primaryXAxis: CategoryAxis(
                majorGridLines: MajorGridLines(width: 0),
                majorTickLines: MajorTickLines(width: 0),
                axisLine: AxisLine(width: 0)),
            primaryYAxis:
                NumericAxis(isVisible: false, maximum: data[0].value >= data[1].value ? data[0].value : data[1].value),
            series: [
              ColumnSeries<ChartData, int>(
                  dataSource: data,
                  color: Theme.of(context).hoverColor,
                  xValueMapper: (ChartData sale, _) => sale.month,
                  yValueMapper: (ChartData sale, _) => sale.value,
                  width: 0.8)
            ]));
  }
}

class ChartData {
  final int month;
  final double value;

  ChartData({@required this.month, @required this.value});
}
```
![image](https://user-images.githubusercontent.com/33434500/104913524-9fb69e80-5985-11eb-82c6-fc62d7305c2f.png)

But if I set the NumericAxis isVisible of the primaryYAxis to true, the two bars already appear. But I don't want the Y axis to appear. Can someone help me put the two bars please?

**Comment by Dharanipalani** [comment]
Hi @Luiscampos97,

We have able to replicate the reported issue at our end. We consider it a bug and we will include the fix in the next patch release which will be expected to roll out on mid of the next week. Please be patience until then. Temporarily you can hide the y axis by the following workaround.

`  primaryYAxis: NumericAxis( 
     axisLine: AxisLine(width: 0), 
      labelStyle: TextStyle(fontSize: 0), 
), 
`

Thanks,
Dharanitharan. P

**Comment by LuisCampos97** [comment]
Hi @Dharanipalani,
Its worked! Thank you very much.
I will wait for the fix too then.

Thanks,
Luís

**Comment by Dharanipalani** [comment]
Hi @LuisCampos97,

We are happy to inform you that the reported issue has been fixed in our latest version **v18.4.41**.Now the column series is visible even if the **y-axis visible** is set to **false**. Please upgrade our chart package to the latest version from the following link.

Latest [version:]( https://pub.dev/packages/syncfusion_flutter_charts/versions/18.4.41)

Thanks,
Dharanitharan. P
